qiskit_ibm_runtime/accounts/account.py

```diff
@@ -278,9 +282,13 @@ def __init__(
             proxies: Proxy configuration.
             verify: Whether to verify server's TLS certificate.
             channel_strategy: Error mitigation strategy.
+            private_endpoint: Connect to private API URL.
         """
         super().__init__(token, instance, proxies, verify, channel_strategy)
-        resolved_url = url or IBM_CLOUD_API_URL
+        if private_endpoint:
+            resolved_url = PRIVATE_EU_API_URL
```

Comment:
Private endpoint should only add the `private` prefix without the region. All regions can have private endpoints, so it should not be attached to any region by default.